setup.py

```diff
@@ -4,6 +4,7 @@
 """
 from os import chdir
 from os.path import abspath, dirname, normpath
+
```

Comment:
I didn't want to touch this, `isort` was complaining about it, and wouldn't shut up.